sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/RetryUtil.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventhubs.implementation;
+
+import com.azure.core.amqp.ExponentialRetryPolicy;
+import com.azure.core.amqp.FixedRetryPolicy;
+import com.azure.core.amqp.RetryOptions;
+import com.azure.core.amqp.RetryPolicy;
+import reactor.core.Exceptions;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+import java.util.Locale;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Helper class to help with retry policies.
+ */
+public class RetryUtil {
+    // So this class can't be instantiated.
+    private RetryUtil() {
+    }
+
+    /**
+     * Given a set of {@link RetryOptions options}, creates the appropriate retry policy.
+     *
+     * @param options A set of options used to configure the retry policy.
+     * @return A new retry policy configured with the given {@code options}.
+     */
+    public static RetryPolicy getRetryPolicy(RetryOptions options) {
+        switch (options.retryMode()) {
+            case FIXED:
+                return new FixedRetryPolicy(options);
+            case EXPONENTIAL:
+                return new ExponentialRetryPolicy(options);
+            default:
+                throw new IllegalArgumentException(
```

Comment:
Use `logger.logAndThrow()`

Reply:
@JonathanGiles 
Since this is a static class and method, I was reading that we shouldn't have static loggers, so I wasn't sure if I should create a static logger for this.
https://azuresdkspecs.z5.web.core.windows.net/JavaSpec.html#general-logging

Reply:
No, just leave as-is for now.